modules/virt-adding-vtpm-to-vm.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-vtpm-devices.adoc
+:_content-type: PROCEDURE
+[id="virt-adding-vtpm-to-vm_{context}"]
+= Adding a vTPM device to a virtual machine
+
+Adding a vTPM device to a virtual machine (VM) stores secrets for that VM.
+
+[NOTE]
+====
+vTPM implementation is non-persistent. vTPM discards stored secrets
+after each shutdown.
+====
+
+.Procedure
+
+. Edit the YAML manifest to add a vTPM device.
++
+[source,YAML]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-windows
+  name: vmi-windows
+spec:
+  domain:
+    devices:
+      tpm: {} <1>
```

Comment:
Is this `tpm: {}` line really the only thing that the user needs to add? (If not, please add example values.)

Reply:
Correct, that is all, there are no options *yet*.
In the future, this is where we'll do things like enable persistence.